packages/utils/src/build-context.js

```diff
@@ -233,10 +233,10 @@ function getAuthor(hash = 'HEAD') {
  * @param {string} hash
  * @return {string}
  */
-function getAvatarUrl(hash = 'HEAD') {
+function getAuthorEmail(hash = 'HEAD') {
   const envHash = getEnvVarIfSet([
     // Manual override
-    'LHCI_BUILD_CONTEXT__AVATAR_URL',
+    'LHCI_BUILD_CONTEXT__AUTHOR_EMAIL',
   ]);
   if (envHash) return envHash;
```

Comment:
how about this?

```suggestion
  if (envHash) return envHash;
  
  // Next try to parse the email from the complete author.
  const author = getAuthor(hash);
  const emailMatch = author.match(/^.* <(\S+@\S+)>$/);
  if (emailMatch) return emailMatch[1];
```

Reply:
I went with something along these lines, but with an error check on the match, and keeping the AVATAR_URL override option available. Might have been overzealous on the tests but I like to be sure I know what a regex is actually doing.